docs/writing-docs/autodocs.md

```diff
@@ -170,6 +233,22 @@ Out of the box, Storybook has a set of components that you can use to customize
 
 ## Troubleshooting
 
+### The table of contents doesn't render as expected
```

Comment:
Should this be under a **Known limitations** section?

Reply:
I went with this approach to surface the issues that we're aware of and clearly mention them to the users without having them buried in that section of the documentation.